### DISCUSSION TITLE
The `/zed` context is needed on assistant panel

### DISCUSSION THREAD

**Discussion by liusong1111** [zed-industries/zed#31691] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

As a newbie to Zed, I want to ask questions on the assistant panel about Zed, like "/zed what's the keyboard shortcut to toggle the Left Dock?" 

The `/zed` should be interpreted as a context for Zed documentation (maybe implemented as a Model Context Server?), and the answer should be more precise.

### Environment

Zed: v0.163.3 (Zed)
OS: Linux Wayland ubuntu 24.04
Memory: 31.2 GiB
Architecture: x86_64
GPU: AMD Radeon RX 580 2048SP (RADV POLARIS10) || radv || Mesa 24.0.9-0ubuntu0.2

**Comment by danilo-leal** [comment]
That's actually a pretty cool ideal! 😃

**Comment by notpeter** [comment]
This could potentially be implemented as a slash command via Zed extension.
